src/calculatorSwitch.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Scanner;
+
+public class calculatorSwitch {
+    public static void main(String[] args) {
+        double firstValue, secondValue, result;
+
+        Scanner input = new Scanner(System.in);
+
+        System.out.println("Please,Input first value");
+        firstValue = input.nextDouble();
+        System.out.println("Please, input operator: : +, -, *, /, %");
+        String operator = input.next();
+        System.out.println("Please,Input second value");
+        secondValue = input.nextDouble();
+
+        switch (operator) {
+            case "+" -> {
```

Comment:
which java version do you use? please write switch as it was on the lecture for java8